.eslintrc.js

```diff
@@ -48,5 +50,12 @@ module.exports = {
         mocha: true,
       },
     },
+    {
+      files: ['assets/scripts/*.js'],
+      env: {
+        browser: true,
+        jquery: true,
+      },
+    },
```

Comment:
I added this to the rubrics PR, so it ended up here as well in the mix. Since it's there, may as well piggyback it here.